sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/ir/expression/literal/DBSPVariantNullLiteral.java

```diff
@@ -37,7 +37,7 @@ public boolean sameValue(@Nullable ISameValue o) {
 
     @Override
     public DBSPLiteral getWithNullable(boolean mayBeNull) {
-        throw new UnimplementedException("getWithNullable");
+        throw new InternalCompilerError("getWithNullable for VariantNullLiteral");
```

Comment:
How does InternalCompilerError show up e2e (e.g., in the WebConsole). Does the error message ask users to file a bug?